jetty-maven-plugin/src/it/jetty-run-mojo-jar-scan-it/MyLibrary/pom.xml

```diff
@@ -3,13 +3,13 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <artifactId>jetty-issue</artifactId>
-    <groupId>org.mehdi</groupId>
+    <groupId>org.eclipse.jetty.its.jetty-run-mojo-jar-scan-it</groupId>
+    <artifactId>jetty-jar-scan</artifactId>
     <version>1.0-SNAPSHOT</version>
```

Comment:
Depending on SNAPSHOT versions is troublesome, don't you think?

Reply:
no the **main** Jetty build it's not depending on snapshot with this :) 
src/it contains projects which are not invoked by maven-invoker-plugin.
The Jetty build do not try to download those projects.